pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,71 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
+
+@pytest.fixture(scope='module')
```

Comment:
this is not necessary, function scoped is fine

Reply:
:+1: commit upcoming